staking_deposit/utils/constants.py

```diff
@@ -14,8 +14,8 @@
 ETH1_ADDRESS_WITHDRAWAL_PREFIX = bytes.fromhex('01')
 
 ETH2GWEI = 10 ** 9
-MIN_DEPOSIT_AMOUNT = 2 ** 0 * ETH2GWEI
-MAX_DEPOSIT_AMOUNT = 2 ** 5 * ETH2GWEI
+MIN_DEPOSIT_AMOUNT = 1000 * ETH2GWEI
```

Comment:
Is there an advantage to using 1k instead of 35k here?

Reply:
It is a min deposit amount. If we define it as 35k (which is equal to max deposit amount), then the validators can only deposit a 35k amount at once.